app/src/lib/stores/app-store.ts

```diff
@@ -2154,18 +2160,37 @@ export class AppStore extends TypedBaseStore<IAppState> {
   /** starts the conflict resolution flow, if appropriate */
   private async showMergeConflictsDialog(
     repository: Repository,
-    conflictState: MergeConflictState
+    conflictState: MergeConflictState,
+    multiCommitOperationState: IMultiCommitOperationState | null,
+    tip: Tip
   ) {
+    if (multiCommitOperationState === null && tip.kind === TipState.Valid) {
```

Comment:
I'd say this `if` deserves a comment explaining why merge is "special" in this regard and it needs to be "initialized" here.

Reply:
It isn't special actually it is just the only one of cherry-picking and rebase we have implemented so far. Any of them once using the multi commit operation will require initialization here.

Reply:
Cool! I thought this was merge-specific. ✅